src/RecoveryServices/RecoveryServices.Backup/RecoveryServices.Backup.csproj

```diff
@@ -12,7 +12,9 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.Management.RecoveryServices.Backup" Version="3.0.1-preview" />
+    <PackageReference Include="Microsoft.Azure.Management.RecoveryServices.Backup" Version="3.1.1-preview" />
+    <PackageReference Include="Microsoft.Rest.ClientRuntime" Version="2.3.19" />
+    <PackageReference Include="Microsoft.Rest.ClientRuntime.Azure" Version="3.3.19" />
   </ItemGroup>
```

Comment:
Please remove all references of `ClientRuntime` packages  from all your `.csproj` files. 

This `Import` adds all common deps for you: https://github.com/Azure/azure-powershell/blob/0322f7f3a336468566dd13249866be98eda2a47b/src/RecoveryServices/RecoveryServices.Backup/RecoveryServices.Backup.csproj#L7